boat-scaffold/src/main/java/com/backbase/oss/codegen/java/BoatSpringCodegenParameter.java

```diff
@@ -0,0 +1,142 @@
+        super.isMap = codegenParameter.isMap;
+        super.isFile = codegenParameter.isFile;
+        super.isEnum = codegenParameter.isEnum;
+        super.isEnumRef = codegenParameter.isEnumRef;
+        super._enum = codegenParameter._enum;
+        super.allowableValues = codegenParameter.allowableValues;
+        super.items = codegenParameter.items;
+        super.additionalProperties = codegenParameter.additionalProperties;
+        super.vars = codegenParameter.vars;
+        super.requiredVars = codegenParameter.requiredVars;
+        super.mostInnerItems = codegenParameter.mostInnerItems;
+        super.vendorExtensions = codegenParameter.vendorExtensions;
+        super.hasValidation = codegenParameter.hasValidation;
+        super.isNullable = codegenParameter.isNullable;
+        super.isDeprecated = codegenParameter.isDeprecated;
+        super.required = codegenParameter.required;
+        super.maximum = codegenParameter.maximum;
+        super.exclusiveMaximum = codegenParameter.exclusiveMaximum;
+        super.minimum = codegenParameter.minimum;
+        super.exclusiveMinimum = codegenParameter.exclusiveMinimum;
+        super.maxLength = codegenParameter.maxLength;
+        super.minLength = codegenParameter.minLength;
+        super.pattern = codegenParameter.pattern;
+        super.maxItems = codegenParameter.maxItems;
+        super.minItems = codegenParameter.minItems;
+        super.uniqueItems = codegenParameter.uniqueItems;
+        super.multipleOf = codegenParameter.multipleOf;
+        super.isNull = codegenParameter.isNull;
+        super.isVoid = codegenParameter.isVoid;
+        super.setAdditionalPropertiesIsAnyType(codegenParameter.getAdditionalPropertiesIsAnyType());
+        super.setHasVars(codegenParameter.getHasVars());
+        super.setSchema(codegenParameter.getSchema());
+        super.setUniqueItemsBoolean(codegenParameter.getUniqueItemsBoolean());
+        super.setMaxProperties(codegenParameter.getMaxProperties());
+        super.setMinProperties(codegenParameter.getMinProperties());
+        super.setHasRequired(codegenParameter.getHasRequired());
+        super.setHasDiscriminatorWithNonEmptyMapping(codegenParameter.getHasDiscriminatorWithNonEmptyMapping());
+        super.setComposedSchemas(codegenParameter.getComposedSchemas());
+        super.setHasMultipleTypes(codegenParameter.getHasMultipleTypes());
+        super.setContent(codegenParameter.getContent());
+        super.setRequiredVarsMap(codegenParameter.getRequiredVarsMap());
+        super.setRef(codegenParameter.getRef());
+        super.setAdditionalProperties(codegenParameter.getAdditionalProperties());
+        if (StringUtils.contains(dataType, ".")) {
+            packageName = StringUtils.substringBeforeLast(dataType, ".") + ".";
+            simpleName = StringUtils.substringAfterLast(dataType, ".");
+        } else {
+            packageName = "";
+            simpleName = datatypeWithEnum;
```

Comment:
Enums are not used in API parameters (compare api.mustache to pojo.mustache). Before this PR `collectionDataTypeParam` was using `dataType` not `dataTypeWithEnum`